webdriver/tests/new_session/response.py

```diff
@@ -1,29 +1,27 @@
 import uuid
 import pytest
 
-from six import string_types
-
 from tests.support.asserts import assert_success
 
 
 def test_sessionid(new_session, add_browser_capabilities):
     response, _ = new_session({"capabilities": {"alwaysMatch": add_browser_capabilities({})}})
     value = assert_success(response)
-    assert isinstance(value["sessionId"], string_types)
+    assert isinstance(value["sessionId"], str)
     uuid.UUID(hex=value["sessionId"])
 
 
 @pytest.mark.parametrize("capability, type", [
-    ("browserName", string_types),
-    ("browserVersion", string_types),
-    ("platformName", string_types),
+    ("browserName", str),
```

Comment:
This changes the names of some tests, and explains some of the noise in https://wpt.fyi/results/webdriver/tests/new_session/response.py?diff&filter=ADC&run_id=5736693317500928&run_id=5762886511099904

The new names are better.